### PR TITLE
Rate Translations - inconsistent behaviour

### PR DESCRIPTION
**Pull request by ealbu**
Send message without improvement if we already have a rating or a feedback message

### CODE REVIEW

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs [hunk 1]
```diff
@@ -247,6 +247,25 @@ private void EditorController_ActiveDocumentChanged(object sender, DocumentEvent
...
 
+		/// <summary>
+		/// We are doing this when the user forces the sending of the feedback(clicks on SendFeedback) without any improvement, rating or feedback message
+		/// because a feedback cannot be sent without any info so we're adding the original target itself as a suggestion
+		/// </summary>
+		/// <param name="segmentId">When this is null the user clicked on SendFeedback instead of it being sent automatically</param>
+		/// <param name="suggestion"></param>
+		/// <param name="rating"></param>
+		/// <returns></returns>
+		private string EnsureFeedbackWillGetThrough(SegmentId? segmentId, ImprovementFeedback suggestion, dynamic rating)
```
Comment (andreaGhisa) [inline SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs:258]: Is it possible to remove suggestion != null and use suggestion?.Improvement == null

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs [hunk 2]
```diff
@@ -247,6 +247,25 @@ private void EditorController_ActiveDocumentChanged(object sender, DocumentEvent
 			ActiveDocument.ActiveSegmentChanged += ActiveDocument_ActiveSegmentChanged;
 		}
 
+		/// <summary>
+		/// We are doing this when the user forces the sending of the feedback(clicks on SendFeedback) without any improvement, rating or feedback message
+		/// because a feedback cannot be sent without any info so we're adding the original target itself as a suggestion
+		/// </summary>
+		/// <param name="segmentId">When this is null the user clicked on SendFeedback instead of it being sent automatically</param>
+		/// <param name="suggestion"></param>
```
Comment (andreaGhisa) [inline SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs:255]: I the description for the parameters is empty please remove them. It adds unnecessary lines to the class

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Events/ShouldSendFeedbackEventHandler.cs [hunk 3]
```diff
@@ -2,5 +2,5 @@
```
Comment (andreaGhisa) [inline SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Events/ShouldSendFeedbackEventHandler.cs:2]: Should we increase the last digit from a manifest file? I think we already send a version for testing.

**Review by andreaGhisa — CHANGES_REQUESTED** [review]

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs (hunk 1 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs:255]: done

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Events/ShouldSendFeedbackEventHandler.cs (hunk 3 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Events/ShouldSendFeedbackEventHandler.cs:2]: done

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs (hunk 2 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs]: done

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs (hunk 1 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs:255]: done

**Review by andreaGhisa — APPROVED** [review]

**Pull request merged**